redfish-core/include/license_messages.hpp

```diff
@@ -0,0 +1,146 @@
+
+#include "http_response.hpp"
+
+#include <nlohmann/json.hpp>
+
+namespace redfish
+{
+
+namespace messages
+{
+
+static void addMessageToErrorJson(nlohmann::json& target,
```

Comment:
Why did we need to copy these from https://github.ibm.com/openbmc/bmcweb/blob/e3a01d08e38ef53ef926e5006ad710264ac233c2/redfish-core/src/error_messages.cpp#L25 ?

Reply:
those are statice function there so have copied here as well
do you suggest to make use of them instead of one more copy

Reply:
done. removed function in license_messages.hpp file and used already existing function  error_messages.cpp